mathjax3-ts/output/common/FontData.ts

```diff
@@ -428,9 +441,9 @@ export class FontData {
      */
     public createVariant(name: string, inherit: string = null, link: string = null) {
         let variant = {
-            linked: [] as CharMap[],
-            chars: (inherit ? Object.create(this.variant[inherit].chars) : {}) as CharMap
-        };
+            linked: [] as CharMap<C>[],
+            chars: (inherit ? Object.create(this.variant[inherit].chars) : {}) as CharMap<C>
+        } as V;
```

Comment:
Would that work by writing `variant: V = `. Not sure if that will coerce the type, but it would look cleaner.

Reply:
No, it doesn't do the casting.  I'd still need the `as V`.